File: 0009-palindrome-number/solution.c
bool isPalindrome(int x) {
    if(x<0){
        return false;
    }
    long long temp2 = x;
    long long temp = 0;
    while(x>0){
        temp = temp*10 +(x % 10);
        x = x/10;
    }
    if(temp == temp2){
        return true;
    }
    else{
        return false;
    }
}

File: 0009-palindrome-number/solution.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0){
            return false;
        }

        int temp = x;
        int number = 0;

        while(x>0){
            number = number * 10  + x % 10;
            x /= 10;  
        }


        if(number == temp){
            return true;
        }
        return false;    
    }
}
